xml/decoders/tcs/CV1_CV99_wow.xml

```diff
@@ -768,19 +768,22 @@
 	<label xml:lang="it">Gruppo Effetti filo Bianco</label>
   </variable>
   <variable CV="49" mask="XXXXVVVV" item="Output 1 effect generated">
-        <qualifier>
+    <qualifier>
 	  <variableref>White Wire Effect Group</variableref>
 	  <relation>eq</relation>
 	  <value>1</value>
 	</qualifier>
```

Comment:
The start tag and the end tag doesn't start on the same column after your changes when viewing the file on GitHub. It seems that you have a different TAB setting than GitHub and that you "fixed" spacing for yourself but made it worse when viewing at GitHub.
```
    <qualifier>
	</qualifier>
```
This is because TAB characters on the line of the end tag. If you want, you could change the TABs to spaces to address this problem.

This comment applies to multiple cases in this file.